SimTracker/SiPhase2Digitizer/plugins/PixelDigitizerAlgorithm.h

```diff
@@ -52,10 +52,10 @@ class PixelDigitizerAlgorithm : public Phase2TrackerDigitizerAlgorithm {
   void add_cross_talk(const Phase2TrackerGeomDetUnit* pixdet) override;
 
   // Addition four xtalk-related parameters to PixelDigitizerAlgorithm specific parameters initialized in Phase2TrackerDigitizerAlgorithm
-  const double odd_row_interchannelCoupling_next_row_;
-  const double even_row_interchannelCoupling_next_row_;
-  const double odd_column_interchannelCoupling_next_column_;
-  const double even_column_interchannelCoupling_next_column_;
+  double odd_row_interchannelCoupling_next_row_;
+  double even_row_interchannelCoupling_next_row_;
+  double odd_column_interchannelCoupling_next_column_;
+  double even_column_interchannelCoupling_next_column_;
```

Comment:
Is this your intension to keep all of these doubles while you are using only one of them. I see you mostly comment them out in 
https://github.com/cms-sw/cmssw/blob/2698506adb68a81503769ceb892b0ff476ff0941/SimTracker/SiPhase2Digitizer/plugins/PixelBrickedDigitizerAlgorithm.cc#L28-L40

This is considered to be a code clean up, maybe we can keep them for now.

Reply:
@srimanob these parameters describe the most generic cross-talk between neighboring channels
Fortunately, lab measurements show a simpler picture (so far...)